ext/web/02_quota_exceeded_error.js

```
@@ -0,0 +1,53 @@
+
+// @ts-check
+/// <reference path="../../core/internal.d.ts" />
+/// <reference path="../../core/lib.deno_core.d.ts" />
+/// <reference path="../webidl/internal.d.ts" />
+/// <reference path="../web/internal.d.ts" />
+/// <reference path="../../cli/tsc/dts/lib.deno_web.d.ts" />
+
+import { primordials } from "ext:core/mod.js";
+const {
+  ObjectDefineProperty,
+  Symbol,
```

Biome (2.1.2)

[error] Do not shadow the global "Symbol" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)